widgets/heatmap/heatmap.go

```diff
@@ -149,5 +323,31 @@ func (hp *HeatMap) Options() widgetapi.Options {
 // The color range is in Xterm color, from 232 to 255.
 // Refer to https://jonasjacek.github.io/colors/.
 func (hp *HeatMap) getCellColor(value float64) cell.Color {
-	return cell.ColorDefault
+	const colorNum = 23
+	scale := hp.maxValue - hp.minValue
+	rgb := int(255 - ((value - hp.minValue) / scale * colorNum))
+	return cell.ColorNumber(rgb)
+}
+
+// initLabels return initial labels, like '0', '1', '2', ...
+func initLabels(l int) []string {
+	var ret []string
+	for i := 0; i < l; i++ {
+		ret = append(ret, fmt.Sprintf("%d", i))
+	}
+	return ret
+}
+
+// minMax returns the min and max values in given integer array.
```

Comment:
Can we potentially use one of the functions defined in the `numbers` package?

https://github.com/mum4k/termdash/blob/74d9de20cbfdc89dc6255bc5a27f3335d0a6fe99/private/numbers/numbers.go#L99-L103

If not, can we modify or contribute another function to that package instead? That way these files can focus on just code directly related to the HeatMap.

Reply:
Didn't notice the `numbers` package, contributed a new function `MinMaxOfMatrix` to it.